JXHTTP/JXHTTPOperation.h

```diff
@@ -303,4 +303,16 @@ typedef NSURLRequest * (^JXHTTPRedirectBlock)(JXHTTPOperation *operation, NSURLR
  */
 + (void)setNetworkActivityIndicatorManager:(id <JXNetworkActivityIndicatorManager>)networkActivityIndicatorManager;
 
+#if __IPHONE_OS_VERSION_MIN_REQUIRED >= __IPHONE_2_0
+
+/**
+ *  The number of active operations.
+ *  This is used to determine whether or not the activity indicator of the device is shown.
```

Comment:
By that same logic, why is this library even specifying what this property is used to determine? :trollface: Caller can do whatever they want with it.